events: change status of `event.returnvalue` to legacy (#46175)
nodejs/node: doc/api/events.md

added: v14.5.0
-->

> Stability: 3 - Legacy: Use [`event.defaultPrevented`][] instead.

* Type: {boolean} True if the event has not been canceled.

> Returns a value which is opposite of the value returned by `defaultPrevented`.

Reviewer: ```suggestion
The value of `event.returnValue` is always the opposite of `event.defaultPrevented`.
```

Author: Applied your suggestion. Thank you for suggestion.
